index.html

```diff
@@ -64,6 +64,115 @@ <h2>Bits and Pieces </h2>
+        </div>
+      
+        </div>
+        <div class="bookSection2">
+            <div class="book">
+                <img src="https://prodimage.images-bn.com/lf?set=key%5Bresolve.pixelRatio%5D,value%5B1%5D&set=key%5Bresolve.width%5D,value%5B300%5D&set=key%5Bresolve.height%5D,value%5B10000%5D&set=key%5Bresolve.imageFit%5D,value%5Bcontainerwidth%5D&set=key%5Bresolve.allowImageUpscaling%5D,value%5B0%5D&set=key%5Bresolve.format%5D,value%5Bwebp%5D&product=path%5B/pimages/9780385348744_p0_v2%5D&call=url%5Bfile:common/decodeProduct.chain%5D"/>
+                <h2>The Demon of Unrest</h2>
+                <p>by Erik Larson</p>
+                <div id="reviewSection" class="review-section"> <input type="text" id="writeReview" placeholder="Write your review here..."></div>
+            </div>
+            <div class="book">
+                <img src="https://prodimage.images-bn.com/lf?set=key%5Bresolve.pixelRatio%5D,value%5B1%5D&set=key%5Bresolve.width%5D,value%5B300%5D&set=key%5Bresolve.height%5D,value%5B10000%5D&set=key%5Bresolve.imageFit%5D,value%5Bcontainerwidth%5D&set=key%5Bresolve.allowImageUpscaling%5D,value%5B0%5D&set=key%5Bresolve.format%5D,value%5Bwebp%5D&product=path%5B/pimages/9781538756317_p0_v2%5D&call=url%5Bfile:common/decodeProduct.chain%5D"/>
+                <h2>Think Twice</h2>
+                <p>by Harlan Coben</p>
+                <div id="reviewSection" class="review-section"> <input type="text" id="writeReview" placeholder="Write your review here..."></div>
+            </div>
+            <div class="book">
+                <img src="https://prodimage.images-bn.com/lf?set=key%5Bresolve.pixelRatio%5D,value%5B1%5D&set=key%5Bresolve.width%5D,value%5B300%5D&set=key%5Bresolve.height%5D,value%5B10000%5D&set=key%5Bresolve.imageFit%5D,value%5Bcontainerwidth%5D&set=key%5Bresolve.allowImageUpscaling%5D,value%5B0%5D&set=key%5Bresolve.format%5D,value%5Bwebp%5D&product=path%5B/pimages/9781250178633_p0_v3%5D&call=url%5Bfile:common/decodeProduct.chain%5D"/>
+                <h2>The Women</h2>
+                <p>by Kristin Hannah</p>
+                <div id="reviewSection" class="review-section"> <input type="text" id="writeReview" placeholder="Write your review here..."></div>
+            </div>
+          
+            </div>
+            <div class="bookSection2">
+                <div class="book">
+                    <img src="https://prodimage.images-bn.com/lf?set=key%5Bresolve.pixelRatio%5D,value%5B1%5D&set=key%5Bresolve.width%5D,value%5B300%5D&set=key%5Bresolve.height%5D,value%5B10000%5D&set=key%5Bresolve.imageFit%5D,value%5Bcontainerwidth%5D&set=key%5Bresolve.allowImageUpscaling%5D,value%5B0%5D&set=key%5Bresolve.format%5D,value%5Bwebp%5D&product=path%5B/pimages/9780593536131_p0_v1%5D&call=url%5Bfile:common/decodeProduct.chain%5D"/>
+                    <h2>The Backyard Bird Chronicles</h2>
```

Comment:
looks great, but its all hard coded , must be an easier and clean way to add.

as i see write now, most of those divs share the same style , only url of the image is dynamic.

I will isolate one div as a component / function and pass only bookInfo object as params for instance 
Const bookInfos = [{url: "lalala", title: "llalala", description:"lalalala"}, {url: "lalala", title: "llalala", description:"lalalala"}, {url: "lalala", title: "llalala", description:"lalalala"}, {url: "lalala", title: "llalala", description:"lalalala"},]
Function bookComponent ( bookInfos ) {

forEach.bookInfos( bookInfo => 
   <div class="book">
       <img scr={bookInfo.url}>
       <p>{bookInfo.title}>
       <p>{bookInfo.description}</p>
       </div>
       }
   
      }

Reply:
Sure, I understand. We have to change the whole approach. I will suggest to Team add the whole book content through JS, and make it more dynamic.